libs/dh/admin/feature-invite-user-modal/src/lib/dh-assignable-user-roles/dh-assignable-user-roles.component.html

```diff
@@ -0,0 +1,78 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+﻿<ng-container *transloco="let t; read: 'admin.userManagement.inviteUser'">
+  <ng-container
+    *rxLet="
+      assignableUserRoles$;
+      let assignableUserRoles;
+      error: error;
+      suspense: suspense;
+      errorTrigger: hasGeneralError$
+    "
+  >
+    <watt-card variant="watt-solid">
+      <watt-card-title>
+        <ng-container
+          *ngTemplateOutlet="
+            userRolesHeadline;
+            context: {
+              $implicit: assignableUserRoles.length
+            }
+          "
+        ></ng-container>
+      </watt-card-title>
+      <div class="userRoles">
+        <ul class="watt-text-s">
+          <li *ngFor="let userRole of assignableUserRoles">
+            <ng-container>
```

Comment:
nitpick: the `ng-container` element can be removed since it's not used for anything.